samples/utils.js

```diff
@@ -47,12 +47,16 @@ window.chartColors = {
 			this._seed = seed;
 		},
 
-		rand: function(min, max) {
+		randF: function(min, max) {
 			var seed = this._seed;
 			min = min === undefined ? 0 : min;
 			max = max === undefined ? 1 : max;
 			this._seed = (seed * 9301 + 49297) % 233280;
-			return min + (this._seed / 233280) * (max - min);
+			return () => min + (this._seed / 233280) * (max - min);
```

Comment:
can you elaborate on the change to return a function instead?

Reply:
will be changed in https://github.com/datavisyn/chartjs-chart-box-and-violin-plot/pull/16/files again